framework/src/main/java/org/checkerframework/common/returnsreceiver/DescribeImages.astub

```diff
@@ -0,0 +1,24 @@
+// Support for the EC2 classes that are automatically handled by the Called Methods Checker.
```

Comment:
@kelloggm just confirming, by placing this `.astub` file in the same directory as the source code, I think the checker jar will include the stub file and it will automatically get loaded without passing an `-Astubs` option.  Is this correct?

Reply:
Combined with the `@StubFiles` annotation that I added on `ReturnsReceiverChecker`, yes.